chapter-01/recipe-04/README.md

```diff
@@ -1,5 +1,6 @@
 # Controlling compilation with conditionals
 
-Abstract to be written ...
+This recipe demonstrates how to use the `if-else-endif` conditional construct in `CMakeLists.txt`.
```

Comment:
Could be nice to link to the doc when appropriate.
https://cmake.org/cmake/help/latest/command/if.html

don't you think?